Cargo.toml

```diff
@@ -42,6 +42,7 @@ thread_profiler = { version = "0.1", optional = true }
 [dev-dependencies]
 cgmath = "0.14"
 genmesh = "0.4"
+amethyst_animation = { path = "amethyst_animation", version = "0.1.0" }
```

Comment:
So you decided not to reexport this package? Should we maybe do the same for the `gltf` crate then, which also doesn't seem like a "mandatory" crate?

Reply:
Yeah, we should.